src/NewPost.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+
+export class NewPost extends Component {
+  state = {
+    title: "",
+    author: "",
+    createdOn: "",
+    body: "",
+    tags: []
+  };
+
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+  };
+
+  render() {
+    return (
+      <form>
+        <input
+          type="text"
+          name="title"
+          placeholder="blog title"
+          value={this.state.title}
+          onChange={e => this.handleChange(e)}
+        />
+        <br />
+        <input
+          type="text"
+          name="author"
+          placeholder="author"
+          value={this.state.author}
+          onChange={e => this.handleChange(e)}
+        />
+        <br />
+        <input
```

Comment:
`createdOn` should probably get set automatically when the form is submitted, similar to `ID`. it'd be nice to let the author avoid manually setting a date. getting the current date in the right format of `YYYY-MM-DD` would take some wrangling with the JS `Date` object, though, so for now maybe just keep this input or leave it out.